src/modules/InviteUser/components/InviteUser.vue

```diff
@@ -0,0 +1,379 @@
+import BaseInput from "@/components/BaseInput.vue";
+import BaseAlert from "@/components/BaseAlert.vue";
+import { inviteUser } from "../services";
+export default {
+  name: "InviteUser",
+  data() {
+    return {
+      name: {
+        value: "",
+        error: "",
+      },
+      email: {
+        value: "",
+        error: "",
+      },
+      role: {
+        value: "",
+      },
+      submission: {
+        message: "",
+        isVerified: false,
+      },
+    };
+  },
+  components: {
+    BaseAlert,
+  },
+  methods: {
+    formData() {
+      return {
+        email: this.email.value,
+        name: this.name.value,
+        role: this.role.value,
+      };
+    },
+    validateField(field) {
+      if (field === "name") {
+        let response = validateUserName(this.name.value);
+        this.name.error = response.errorMessage;
+      }
+      if (field === "email") {
+        let response = validateEmail(this.email.value);
+        this.email.error = response.errorMessage;
+      }
+    },
+    async handleSubmit() {
+      console.log(this.role.value);
+      if (!this.name.value || !this.email.value || !this.role.value) {
+        return (this.submission.message = "Field must not be empty");
+      }
+      if (this.name.error || this.email.error) {
+        return (this.submission.message =
+          "Some fields are not filled properly");
+      }
+      try {
+        this.submission.isVerified = true;
+        const response = await inviteUser(this.formData());
+      } catch (err) {
+        this.submission.message = err;
+      }
+    },
+  },
+};
+</script>
+
+<template>
+  <!-- Button trigger modal -->
+  <button
+    type="button"
+    class="btn invite-button"
+    data-cy="invite-btn"
+    data-bs-toggle="modal"
+    data-bs-target="#exampleModal"
+  >
+    Invite member
+  </button>
+  <!-- Modal -->
+  <div
+    class="modal fade"
+    id="exampleModal"
+    tabindex="-1"
+    aria-labelledby="exampleModalLabel"
+    aria-hidden="true"
+  >
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="add-a-dialog-box screen">
+          <div class="title">
+            <div class="title-left">
+              <div class="header-frame">
+                <img
+                  src="../../../assets/plus-stroke.png"
+                  alt=""
+                  class="plus-stroke"
+                />
+                <div class="invite-new-member">Invite new member</div>
+              </div>
+              <p class="add-a-new-member-to-proshore">
+                Add a new member to proshore
+              </p>
+            </div>
+            <div class="close-button">
+              <button
+                class="close-rectangle"
+                type="button"
+                data-cy="close-invite-btn"
+                data-bs-dismiss="modal"
+              >
+                <img src="../../../assets/x-lg.png" alt="" class="x-lg" />
```

Comment:
import image at the <script> tag and then use

Reply:
@anantKC

Reply:
the images will be replaced by icons